satellite/modules/vm_manage.py

```diff
@@ -31,20 +49,39 @@ def isOnline(host):
 
 
 def virsh_start_vm(vm_name, com_ip):
+    """Starts the virtual machine on remote system
+    :param vm_name: Name of virtual machine
+    :param com_ip: Compute IP on which virtual machine is running
+
+    :return: start_vm_flag
```

Comment:
above you have written **returns** here you wrote **return**
choose one

Reply:
use return instead of returns